### PR TITLE
chore: update pupperteer and sharp npmmirror 

### PR DESCRIPTION
**Pull request by poyiding**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [x] Other (about what?)

### 💡 Background and solution
https://zhuanlan.zhihu.com/p/637107614

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     slove npm install pupperteer download fail problem      |
| 🇨🇳 Chinese |   解决npm 安装 pupperteer  失败       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 67f48ef</samp>

Add a new environment variable to `.npmrc` to use a mirror site for puppeteer installation. This improves the testing workflow and reduces flakiness.

### <samp>🤖 Generated by Copilot at 67f48ef</samp>

*  Add `PUPPETEER_DOWNLOAD_BASE_URL` environment variable to `.npmrc` to use mirror site for puppeteer installation ([link](https://github.com/ant-design/ant-design/pull/44727/files?diff=unified&w=0#diff-e813096d69c49812e40e00be9ac8fa14d77f0ec1f97ab4c45cb096b3bedd35deL1-R3))

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -1 +1,3 @@
-package-lock=false
\ No newline at end of file
+package-lock=false
+
+PUPPETEER_DOWNLOAD_BASE_URL="https://cdn.npmmirror.com/binaries/chrome-for-testing"
```
Comment (yoyo837) [inline .npmrc:3]: 不知道阿里云镜像对国外是否有好 @fengmk2

Comment (poyiding) [reply .npmrc:3]: https://zhuanlan.zhihu.com/p/637107614
puppeteer 资源地址更换了，先在这解决， @fengmk2 阿里云镜像那边解决更换，如果处理在这回复一下，可以去掉。

Comment (fengmk2) [reply .npmrc:3]: 这个就是 npmmirror 的镜像去重新适配的，最新的都重 chrome-for-testing 下载了。
我们还提供了一个给人类浏览的页面 https://registry.npmmirror.com/binary.html?path=chrome-for-testing/

阿里云镜像具体是指哪个？

Comment (fengmk2) [reply .npmrc:3]: > 不知道阿里云镜像对国外是否友好 @fengmk2

npmmirror cdn 走的是阿里云全球加速，飞快的。

**Comment by fengmk2** [comment]
https://github.com/cnpm/binary-mirror-config/pull/47 puppeteer >= 20 和 <= 19 的兼容逻辑

File: .npmrc (hunk 1 above)
Comment (poyiding) [reply .npmrc:3]: 我不太懂的是，antd 里的 puppeteer 大于 19 ，按理 npm install  puppeteer 会从 PUPPETEER_DOWNLOAD_BASE_URL 这里配的下载资源就不会失败，但是现在是从 19 版本的 puppeteer-download-host="https://cdn.npmmirror.com/binaries" 这里下载，导致失败。

阿里云镜像指的是 npmmirror 习惯了老的叫法哈

**Comment by poyiding** [comment]
> 



> [cnpm/binary-mirror-config#47](https://github.com/cnpm/binary-mirror-config/pull/47) puppeteer >= 20 和 <= 19 的兼容逻辑

懂了，谢谢苏大

**Comment by yoyo837** [comment]
我发现在Windows上全新安装， 还有个二进制包 `harp-libvips` 需要设置镜像 `npm_config_sharp_libvips_binary_host`， 当前PR接受的话，一起考虑一下它？

**Comment by poyiding** [comment]
> 我发现在Windows上全新安装， 还有个二进制包 `harp-libvips` 需要设置镜像 `npm_config_sharp_libvips_binary_host`， 当前PR接受的话，一起考虑一下它？

 应该是sharp 依赖了 libvips，我没有 windows 环境，需要你验证一下哈

**Review by MadCcc — APPROVED** [review]

**Pull request merged**